CONTRIBUTING.rst

```diff
@@ -0,0 +1,678 @@
+locally requires significant setup. You can set up Travis CI in your fork of
+Airflow by following the `Travis
+CI Getting Started
+guide <https://docs.travis-ci.com/user/getting-started/>`__..
+
+
+There are two different options available for running Travis CI, and they are
+set up on GitHub as separate components:
+
+-   **Travis CI GitHub App** (new version)
+-   **Travis CI GitHub Services** (legacy version)
+
+Using Travis CI GitHub App (new version)
+----------------------------------------
+
+-   Once `installed <https://github.com/apps/travis-ci/installations/new/permissions?target_id=47426163>`__,
+    configure the Travis CI GitHub App at
+    `Configure Travis CI <https://github.com/settings/installations>`__.
+
+-   Set repository access to either "All repositories" for convenience, or "Only
+    select repositories" and choose ``USERNAME/airflow`` in the drop-down menu.
+
+-   Access Travis CI for your fork at `<https://travis-ci.com/USERNAME/airflow>`__.
+
+Using Travis CI GitHub Services (legacy version)
+------------------------------------------------
+
+..
+
+    The apache/airflow project is still using the legacy version.
+
+Travis CI GitHub Services version uses an Authorized OAuth App.
+
+1.  Once installed, configure the Travis CI Authorized OAuth App at
+    `Travis CI OAuth APP <https://github.com/settings/connections/applications/88c5b97de2dbfc50f3ac>`__.
+
+2.  If you are a GitHub admin, click the **Grant** button next to your
+    organization; otherwise, click the **Request** button. For the Travis CI
+    Authorized OAuth App, you may have to grant access to the forked
+    ``ORGANIZATION/airflow`` repo even though it is public.
+
+3.  Access Travis CI for your fork at
+    `<https://travis-ci.org/ORGANIZATION/airflow>`_.
+
+Creating New Projects in Travis CI
+----------------------------------
+
+If you need to create a new project in Travis CI, use travis-ci.com for both
+private repos and open source.
+
+The travis-ci.org site for open source projects is now legacy and you should not use it.
+
+..
+    There is a second Authorized OAuth App available called "Travis CI for Open Source" used
+    for the legacy travis-ci.org service Don't use it for new projects.
+
+More information:
+
+-  `Open Source on travis-ci.com <https://docs.travis-ci.com/user/open-source-on-travis-ci-com/>`__.
+-  `Legacy GitHub Services to GitHub Apps Migration Guide <https://docs.travis-ci.com/user/legacy-services-to-github-apps-migration-guide/>`__.
+-  `Migrating Multiple Repositories to GitHub Apps Guide <https://docs.travis-ci.com/user/travis-migrate-to-apps-gem-guide/>`__.
+
+Changing the Metadata Database
+==============================
+
+When developing features, you may need to persist information to the metadata
+database. Airflow has `Alembic <https://bitbucket.org/zzzeek/alembic>`__. built-in
+module to handle all schema changes. Alembic must be installed on your
+development machine before continuing with migration.
```

Comment:
Elena:  continuing. //continuing with what? "persist information" \> add information ?
Jarek: I updated  this to migration because that's what Alembic creates in this case - it creates a migration script. But it likely needs to be slightly modified to describe this. This is basically a migration script that implements changes to the database and is generated automatically. Maybe alembic's documentation can be better used to explain it.